webapp/tests/Unit/Controller/API/GeneralInfoControllerTest.php

```diff
@@ -2,14 +2,121 @@
 
 namespace App\Tests\Unit\Controller\API;
 
+use App\DataFixtures\Test\SampleSubmissionsFixture;
 use App\Service\DOMJudgeService;
-use App\Tests\Unit\BaseTest;
+use App\Tests\Unit\BaseTest as BaseBaseTest;
 use Generator;
 use Symfony\Component\HttpFoundation\BinaryFileResponse;
 use Symfony\Component\Intl\Countries;
 
 class GeneralInfoControllerTest extends BaseTest
 {
+    private $API_VERSION = 4;
```

Comment:
Why a private var and not a `const`? Because they are public? (note: I think if we later use PHP 7.4+ only consts can also be private)